src/tools/clippy/tests/ui/needless_maybe_sized.fixed

```diff
@@ -1,136 +0,0 @@
-//@aux-build:proc_macros.rs
```

Comment:
If the `.fixed` file is removed, shouldn't the corresponding `.rs` and `.stderr` be removed as well?